src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/Offence.java

```diff
@@ -42,7 +42,7 @@ public class Offence {
     @EqualsAndHashCode
     public static class PK implements Serializable {
         private String code;
-        private Statute statute;
+        private String statute;
```

Comment:
I changed this statute field in the PK class to be a string, otherwise when I tried to save I got this exception
```
11:16:48.602 [ERROR] uk.gov.justice.hmpps.prison.aop.LoggingAspect.logAfterThrowing - Exception in pointcut uk.gov.justice.hmpps.prison.service.reference.OffenceService createOffence()
org.springframework.dao.InvalidDataAccessApiUsageException: Can not set uk.gov.justice.hmpps.prison.repository.jpa.model.Statute field uk.gov.justice.hmpps.prison.repository.jpa.model.Offence$PK.statute to java.lang.String; nested exception is java.lang.IllegalArgumentException: Can not set uk.gov.justice.hmpps.prison.repository.jpa.model.Statute field uk.gov.justice.hmpps.prison.repository.jpa.model.Offence$PK.statute to java.lang.String
```

Reply:
@yasinmustafa Isn't this just missing the `@enumerated` annotation? with String type